api/cluster.go

```diff
@@ -21,6 +21,8 @@ import (
 	"sort"
 	"strings"
 
+	"net/url"
```

Comment:
Please pay attention to imports sorting. I know IDEs tend to mess this up. Please move net/url to the internal packages block.

Reply:
fmt cares with ordering inside groups only. and the linter is useless in this case with its 50 error reports, none of which about the import grouping.